### ISSUE TITLE
there was an error while searching: This IP address has been blocked by YouTube (429).

### ISSUE THREAD

**Issue by vanna-code** [lavalink-devs/Lavalink#675]
how to fix ? `there was an error while searching: This IP address has been blocked by YouTube (429).`

**Comment by Xirado** [comment]
There is literally a [pinned issue](https://github.com/freyacodes/Lavalink/issues/369) about this...

**Comment by devoxin** [comment]
Duplicate of #369
